x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/condition_entry_input/index.test.tsx

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { shallow, mount } from 'enzyme';
+import React from 'react';
+import {
+  ConditionEntry,
+  ConditionEntryField,
+  OperatingSystem,
+} from '../../../../../../../common/endpoint/types';
+
+import { ConditionEntryInput } from '.';
+import { EuiSuperSelectProps } from '@elastic/eui/src/components/form/super_select/super_select';
+
+let onRemoveMock: jest.Mock;
+let onChangeMock: jest.Mock;
+let onVisitedMock: jest.Mock;
+
+const entry: ConditionEntry = {
```

Comment:
Can you make this type `readonly` or use Object.freeze on the value.
Just want to ensure that none trips themselves up if they happen to mutate the value in a test.